### DISCUSSION TITLE
RedisTimeSeries Module for Valkey

### DISCUSSION THREAD

**Discussion by adimishraa7** [valkey-io/valkey#713] [Q&A]
Is there a way to build out  the RedisTimeSeries for Vakey and use the redistimeseries.so for the redis timeseries module or the redistimeseries module is not supported at all for Valkey right now? Can we not use the timeseries module for Valkey?

**Comment by madolson** [comment]
If the version of the redistimeseries.so you are using is compatible with Redis 7.2, it will also be compatible with Valkey 7.2 since we are ABI compatible.

**Reply by m2b** [reply]
@adimishraa7 have you succeeded building it.
